.github/workflows/build-frontend.yml

```diff
@@ -48,12 +48,19 @@ jobs:
           printf '%s' "$ENVFILE" > apps/frontend/.env.local
         env:
           ENVFILE: ${{ vars.ENVFILE }}
-      - name: Build genshin-optimizer
+      - name: Build frontend
         run: |
           NX_URL_GITHUB_GO_CURRENT_VERSION="https://github.com/${{ github.repository }}/commit/$(git rev-parse HEAD)" \
           NX_DAEMON="false" \
           NX_SHOW_DEV_COMPONENTS="${{inputs.show_dev_components}}" \
-          yarn run nx run ${{inputs.frontend_name}}:build:production
+      - name: Build gi-frontend
+        if: inputs.frontend_name == 'gi-frontend'
+        run: yarn nx run gi-frontend:next:build
```

Comment:
Cant you just override build in gi-frontend project.json to call next:build?

Reply:
i had trouble getting the `build:production` to work, this is mostly a bandaid anyways